torch/quantization/fx/quantization_patterns.py

```diff
@@ -367,19 +367,27 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
                 args = load_arg(quantized=quantized_input_idxs)(self.linear_node.args)
                 args = load_arg(quantized=False)(self.linear_node.args)
                 kwargs = load_arg(quantized=False)(self.linear_node.kwargs)
-                linear_out = quantizer.quantized_graph.create_node(
-                    'call_function', torch.nn.functional.linear, args, kwargs)
+                op_out = quantizer.quantized_graph.create_node(
+                    "call_function", torch.nn.functional.linear, args, kwargs)
+                if self.relu_node:
+                    relu_args = [op_out]
+                    relu_args.extend(load_arg(quantized=False)(self.relu_node.args[1:]))
+                    relu_kwargs = load_arg(quantized=False)(self.relu_node.kwargs)
+                    op_out = quantizer.quantized_graph.create_node(
+                        "call_function", torch.nn.functional.relu, tuple(relu_args), relu_kwargs)
+
                 if activation_statically_quantized:
                     # quantize output for statically quantized linear op
                     root_module = quantizer.modules['']
+                    act_post_process_name = self.relu_node.name if self.relu_node else self.linear_node.name
```

Comment:
if debug is true then do we still insert observer at the output of relu node? relu doesn't need output_scale and zero_point, right?

Reply:
yes, we still insert in the output of relu. debug does not affect where observer is inserted, it only affect the quantized model we produce.
relu itself does not need output_scale/output_zero_point, but linear_relu and conv_relu need these.